.github/workflows/conda.yml

```diff
@@ -6,9 +6,10 @@ on:
   pull_request:
     paths:
       - setup.py
-      - requirements.txt
+      - setup.cfg
```

Comment:
Interested in why we would want to run conda build testing on changes to setup.cfg? My understanding is that this would now only impact our flake8 configuration, which on their own I wouldn't think would have an impact on the builds

Reply:
Seems reasonable